### PR TITLE
Create IBM.ibm-commonui-operator.release-1.4.yaml

### PR DESCRIPTION
**Pull request by siddarthpatel**
Enable prow for `release-1.4` common ui operator branch

### CODE REVIEW

File: prow/cluster/jobs/IBM/ibm-commonui-operator/IBM.ibm-commonui-operator.release-1.4.yaml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        - entrypoint
+        - make
+        - test
+        image: quay.io/multicloudlab/build-tool:v20200817-b1f2b4c05
+        name: ""
+        securityContext:
+          privileged: true
+  - name: coverage_ibm-commonui-operator_postsubmit
+    branches:
+    - ^master$
```
Comment (horis233) [inline prow/cluster/jobs/IBM/ibm-commonui-operator/IBM.ibm-commonui-operator.release-1.4.yaml:90]: should be `^release-1.4$`

File: prow/cluster/jobs/IBM/ibm-commonui-operator/IBM.ibm-commonui-operator.release-1.4.yaml [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        - name: CODECOV_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: ibm-commonui-operator-codecov-token
+              key: codecov-token
+        securityContext:
+          privileged: true
+  - name: build_ibm-commonui-operator_postsubmit
+    branches:
+    - ^master$
```
Comment (horis233) [inline prow/cluster/jobs/IBM/ibm-commonui-operator/IBM.ibm-commonui-operator.release-1.4.yaml:110]: should be ^release-1.4$

File: prow/cluster/jobs/IBM/ibm-commonui-operator/IBM.ibm-commonui-operator.release-1.4.yaml [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        - entrypoint
+        - make
+        - build
+        image: quay.io/multicloudlab/build-tool:v20200817-b1f2b4c05
+        name: ""
+        securityContext:
+          privileged: true
+  - name: images_ibm-commonui-operator_postsubmit
+    branches:
+    - ^master$
```
Comment (horis233) [inline prow/cluster/jobs/IBM/ibm-commonui-operator/IBM.ibm-commonui-operator.release-1.4.yaml:125]: should be ^release-1.4$

**Review by morvencao — COMMENTED** [review]
```
duplicated postsubmit job: coverage_ibm-commonui-operator_postsubmit
```

Make sure you're creating prow jobs against correct branch.

**Review by morvencao — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**